Resources/Prototypes/Entities/Objects/Devices/Circuitboards/Machine/production.yml

```diff
@@ -677,3 +677,33 @@
     materialRequirements:
       Steel: 10
       Plasma: 5
+      
+- type: entity
+  id: BoozeDispenserMachineCircuitboard
+  parent: BaseMachineCircuitboard
+  name: booze dispenser machine board
+  description: A machine printed circuit board for a booze dispenser
+  components:
+    - type: Sprite
+      state: service
+    - type: MachineBoard
+      prototype: booze_dispenser
```

Comment:
```suggestion
      prototype: BoozeDispenser
```

Reply:
I saw that. It's inconsistent in the file. But I noticed the majority use this format. So I changed it.